### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: reclama/templates/index.html [hunk 1]
```diff
@@ -14,4 +14,16 @@
       </div>
     </a>
   {% endfor %}
+
+  <div class="archived">Archived events</div>
```
Comment (johngian) [inline reclama/templates/index.html]: Please check if archived events exist. At the moment if no archived events exist it just shows the title.

Comment (johngian) [inline reclama/templates/index.html]: I would prefer this to be initially hidden and have a button to show last archived events. But we can skip this for now and tackle it in another PR. To be futureproof we might also need a list view for all events (like moderator).

**Comment by johngian** [comment]
Looks good to me. r+wc

**Comment by comzeradd** [comment]
Thanks. I did a small update based on your comments. I deal with the rest on a different PR since this will require some extra UI work (eg. pagination).

**Pull request merged**